### PR TITLE
Cat 1210 content tweaks

### PR DESCRIPTION
**Pull request by Olusola-Opencast**

### CODE REVIEW

File: server/views/formPages/ratings/furtherCharges.html [hunk 1]
```diff
@@ -60,7 +59,7 @@ <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
     ]
   }) }}
 
-  {% set guidancePrevious %}<div class="govuk-!-font-weight-bold govuk-!-margin-top-2">You are looking for:</div>
+  {% set guidancePrevious %}<div class="govuk-!-font-weight-bold govuk-!-margin-top-2">Category B might be appropriate if this person has:</div>
```
Comment (kberec) [inline server/views/formPages/ratings/furtherCharges.html:62]: Should this be: "What should I look for?" ?

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy [hunk 2]
```diff
@@ -289,15 +289,15 @@ class LandingPageSpecification extends AbstractSpecification {
     driver.pageSource.contains('This individual was identified as needing a security review, as part of their categorisation, by Another User of LEEDS (HMP) on ' +  LocalDate.now().format('dd/MM/yyyy'))
 
     when: 'the security review page is only saved'
-    securityText << 'security info'
+    securityText << 'security info text'
```
Comment (kberec) [inline integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy:292]: There are quite a few changes like this, when you are changing values in integration test, so it is less generic. 
As a matter of interest, what is the reason behind doing these?
Only asking, as the more files we change more likely we'll have conflicts with other ongoing work.
This PR has loads of refactoring.

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/TasklistRecatSpecification.groovy [hunk 3]
```diff
@@ -83,7 +83,7 @@ class TasklistRecatSpecification extends AbstractSpecification {
 
     fixture.gotoTasklistRecat(false)
     at TasklistRecatPage
-    db.getSecurityData('B2345YZ')[0].status.value == 'PROCESSED'
+   // db.getSecurityData('B2345YZ')[0].status.value == 'PROCESSED'
```
Comment (kberec) [inline integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/TasklistRecatSpecification.groovy:86]: leftover comment?

File: server/views/formPages/ratings/furtherCharges.html (hunk 1 above)
Comment (Olusola-Opencast) [reply server/views/formPages/ratings/furtherCharges.html:62]: corrected

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/TasklistRecatSpecification.groovy (hunk 3 above)
Comment (Olusola-Opencast) [reply integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/TasklistRecatSpecification.groovy]: done

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy (hunk 2 above)
Comment (kberec) [reply integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy:292]: Based on a separate conversation in slack: "Integration test were failing due to minimum character count"

Comment (kkonuganti) [reply integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy:292]: as kornel said we dont need this change. unless we are checking for a specific scenario or specific text.
Technically we should not be checking any content changes through integration tests anyways.

**Review by kberec — APPROVED** [review]

**Review by kkonuganti — APPROVED** [review]

**Pull request merged**